settings/types.go

```diff
@@ -60,3 +60,33 @@ type AirflowPools []struct {
 	PoolSlot        string `yaml:"slots"`
 	PoolDescription string `yaml:"description"`
 }
+
+// types for creating variables and connections yaml files
+
+type VariablesYAML struct {
+	VarYAMLs `mapstructure:"connections" yaml:"variables"`
+}
+
+type VarYAMLs []VarYAML
+
+type VarYAML struct {
+	Key   string `mapstructure:"key" yaml:"key"`
+	Value string `mapstructure:"value" yaml:"value"`
+}
+
+type ConnectionsYAML struct {
+	ConnYAMLs `mapstructure:"connections" yaml:"connections"`
+}
+
+type ConnYAMLs []ConnYAML
+
+type ConnYAML struct {
+	ConnID   string      `mapstructure:"conn_id" yaml:"conn_id"`
+	ConnType string      `mapstructure:"conn_type" yaml:"conn_type"`
+	Host     string      `mapstructure:"host" yaml:"host"`
+	Schema   string      `mapstructure:"schema" yaml:"schema"`
+	Login    string      `mapstructure:"login" yaml:"login"`
+	Password string      `mapstructure:"password" yaml:"password"`
+	Port     int         `mapstructure:"port" yaml:"port"`
+	Extra    interface{} `mapstructure:"extra" yaml:"extra"`
+}
```

Comment:
I don't think so we need `mapstructure` tag for any of these fields

Reply:
not sure